cdc/api/v2/api_helpers.go

```diff
@@ -37,11 +38,19 @@ import (
 	"github.com/r3labs/diff"
 	"github.com/tikv/client-go/v2/oracle"
 	pd "github.com/tikv/pd/client"
+	"go.etcd.io/etcd/client/pkg/v3/logutil"
+	clientv3 "go.etcd.io/etcd/client/v3"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/backoff"
 )
 
+// RegisterImportTaskPrefix denotes the key prefix associated with the entries
+// containning import/restore information in the embedded Etcd of the
+// upstream PD.
+const RegisterImportTaskPrefix = "/tidb/brie/import"
```

Comment:
we should use the exported prefix from tidb package?

Reply:
Then we need to make the package `github.com/pingcap/tidb/br` importable. Upgrading Tiflow to use the latest version  of tidb may involve resolving many dependency issues, as it is currently utilizing v1.1.0-beta.0.20230310043312-53f15f6ed774 of github.com/pingcap/tidb.